.travis.yml

```diff
@@ -7,6 +7,7 @@ before_install:
   - pip install pytest-timeout
 install:
   - pip install -v --global-option=build_ext --global-option="-Itmp-build/include/" --global-option="-Ltmp-build/lib" .
+  - pip install --upgrade pip
```

Comment:
Is the .avro install requiring a newer pip version?
If so, we probably need to document what version is required.
And also move this to before_install prior to any other pip commands.